tests/src/test/resources/confs/validation.conf

```diff
@@ -4,32 +4,31 @@ httpsPort=9595
 tokenListenerPort=9596
 
 [b7a.users]
-[b7a.users.generalUser1]
-password="5baa61e4c9b93f3f0682250b6cf8331b7ee68fd8"
-
-[validationConfig]
-enableRequestValidation = true
-enableResponseValidation = true
+  [b7a.users.generalUser1]
+    password="5baa61e4c9b93f3f0682250b6cf8331b7ee68fd8"
 
 [apikey.tokenConfigs]
-issuer="https://localhost:9095/apikey"
-audience="http://org.wso2.apimgt/gateway"
-certificateAlias="ballerina"
-validateAllowedAPIs=true
+  issuer="https://localhost:9095/apikey"
```

Comment:
Let's remove these configs since we can use the default values for this test case